### DISCUSSION TITLE
Plans for an amd64 version?

### DISCUSSION THREAD

**Discussion by newmy-de** [VictoriaMetrics-Community/homeassistant-addon-victoriametrics#9] [Q&A] [answered]
My homeassistant is running on AMD86 platform. Do you have any Plans for an AMD64 Version on HassOS?
If i exchange the arm-version of the victoria-metrics-prod file in your folder - do you think it will work on my HA?
regards
newmy

**Comment by fuslwusl** [comment, marked as answer]
@newmy-de Sorry for my late answer. Yes, if you exchange the version with the one for your platform it should work. I never tried on amd68 so i am interested if you had success.

**Reply by fuslwusl** [reply]
Thanx to oetken there is an AMD64 version of the addon available now!
